plugins/woocommerce-beta-tester/src/remote-inbox-notifications/index.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * External dependencies
+ */
+import { withDispatch, withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from './data/constants';
+import './data';
+
+function RemoteInboxNotifications( {
+	notifications,
+	deleteNotification,
+	importNotifications,
+	deleteAllNotifications,
+	testNotification,
+	isLoading,
+	notice,
+	setNotice,
+} ) {
+	const importFromEnv = async ( env ) => {
+		const urls = {
```

Comment:
It would be helpful if we could also allow users to use a custom URL for the import. This way, we can test some functionalities unavailable in staging or production and fetch notes from a gist URL to test PR changes. What do you think?

Reply:
Good idea!

Reply:
Added in https://github.com/woocommerce/woocommerce/pull/48735/commits/7e628ac98e11b1031ed01a5f17ac0f10f661c4f1

Reply:
> I encountered an error when trying to delete all notes.

It seems like JN does not like `DELETE` methods. I've changed DELETE to POST. Please try again.